### PR TITLE
Update libs

### PR DESCRIPTION
**Pull request by liujed**
Also fixed a latent bug where timelines always had 0 latency

### CODE REVIEW

**Comment by liujed** [comment]
Depends on akitasoftware/akita-libs#130.

File: rest/learn_client.go [hunk 1]
```diff
@@ -214,6 +215,7 @@ func (c *learnClientImpl) GetUnaggregatedTimeline(ctx context.Context, serviceID
 	q.Add("key", "method")
 	q.Add("key", "path")
 	q.Add("key", "code")
+	q.Add("aggregate", string(api_schema.Aggr_99p))
```
Comment (thatplguy) [inline rest/learn_client.go:218]: Does the capability to get an unaggregated timeline still exist in the new version?  IIRC this meant retrieving one data point per witness, which is presumably why P99 was omitted.

In the new version, this probably returns events at the granularity of the atomic bucket size?

I think that's fine, and in that case it makes sense to add P99 latency.  For individual events, P99 latency wouldn't have made sense.

**Review by thatplguy — APPROVED** [review]

File: rest/learn_client.go (hunk 1 above)
Comment (liujed) [reply rest/learn_client.go:219]: Oh, I entirely missed the intended behaviour of this function. Yeah, it's no longer possible to get an unaggregated timeline. Changed aggregation to 1-minute and fixed the function name in 671427a.

**Comment by liujed** [comment]
Re-requesting review for changes in 671427a.

**Pull request merged**